extensions/exposition/documentation/vary.md

```diff
@@ -0,0 +1,39 @@
+# HTTP request details
+
+The vary directives family provides the capability to include HTTP request details as input for an
+operation call.
+
+## TL;DR
+
+```yaml
+exposition:
+  /:
+    vary:languages: [en, fr]
+    GET:
+      vary:embed:
+        language: lang          # predefined embeddings
+        :x-access-token: token  # raw http header value
+      endpoint: texts.get
+```
+
+## Embeddings
+
+Request parts are embedded into the operation call according to the mapping
+defined by the `vary:embed` directive.
+The keys in the embedding map are the names of the embedding functions, and the values are the
+property names of the operation call input object.
+
+> If a property is already present in the input, the embedded value will overwrite its current
+> value.
+
+### Language
+
+The `language` embedding substitutes the most matching language code based on the `accept-language`
+request header and a list of supported languages defined by the `vary:languages` directive, and also
+adds `accept-language` to the `vary` HTTP response header value.
+
+### Raw headers
+
+Keys in the embedding map starting with a semicolon (:) are the names of HTTP request headers whose
```

Comment:
How it would work with multi-values headers? Like cookies, Accept, etc.

Reply:
Great point. I have to refresh the standard statements.
If any header may have multiple values, then the value of any header embedded should be `string[]`. If multiple values are only possible for a certain list of headers, then only those would be arrays.

Reply:
That would be the best option.